model/actors/miner/task.go

```diff
@@ -82,10 +84,74 @@ func (res *MinerTaskResult) Persist(ctx context.Context, db *pg.DB) error {
 type MinerTaskResultList []*MinerTaskResult
 
 func (ml MinerTaskResultList) PersistWithTx(ctx context.Context, tx *pg.Tx) error {
+	ctx, span := global.Tracer("").Start(ctx, "MinerTaskResultList.PersistWithTx", trace.WithAttributes(label.Int("count", len(ml))))
+	defer span.End()
+
 	for _, res := range ml {
 		if err := res.PersistWithTx(ctx, tx); err != nil {
 			return err
 		}
 	}
 	return nil
 }
+
+// MinerTaskLists allow better batched insertion of Miner-related models.
+type MinerTaskLists struct {
+	MinerInfoModel           MinerInfoList
+	FeeDebtModel             MinerFeeDebtList
+	LockedFundsModel         MinerLockedFundsList
+	CurrentDeadlineInfoModel MinerCurrentDeadlineInfoList
+	PreCommitsModel          MinerPreCommitInfoList
+	SectorsModel             MinerSectorInfoList
+	SectorEventsModel        MinerSectorEventList
+	SectorDealsModel         MinerSectorDealList
+}
```

Comment:
Allows processing all miners in a tipset, collecting all the models and then persisting that. Should help towards making a single Tx per tipset and be faster than using MinerTaskResultList.